code/WorkInProgress/TempEngine.dm

```diff
@@ -763,3 +916,4 @@
 #undef PUMP_POWERLEVEL_5
 #undef LEFT_CIRCULATOR
 #undef RIGHT_CIRCULATOR
+#undef LUBE_CHECK_RATE
```

Comment:
undef MIN_CIRCULATOR_SPEED too

Reply:
Removing MIN_CIRCULATOR_SPEED in favor of property of circulator.